minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloConfigImpl.java

```diff
@@ -47,6 +47,12 @@
  * @since 1.6.0
  */
 public class MiniAccumuloConfigImpl {
+
+  @FunctionalInterface
+  public static interface FinalSiteConfigUpdater {
+    void update(MiniAccumuloConfigImpl cfg);
+  }
```

Comment:
You don't need this interface. Just use `Consumer<MiniAccumuloConfigImpl>`. While it probably doesn't hurt to have an extra interface to select a more expressive name for ease of use, I think the name here, with the word "Final" here is a bit presumptive (what if we need a subsequent modification to support other tests? would we call it "FinalFinal"?). We can just avoid coming up with a name by just using the existing Consumer... also, Consumer has the helpful `andThen` method, which we don't need to reinvent and could be useful.

Reply:
Addressed in f34eb45.